modules/apps/object/object-service/src/main/java/com/liferay/object/internal/model/listener/SystemObjectDefinitionMetadataModelListener.java

```diff
@@ -173,7 +181,9 @@ private String _getExternalModel(T baseModel, long userId)
 			return _jsonFactory.looseSerializeDeep(externalModel);
 		}
 		catch (Exception exception) {
-			_log.error(exception, exception);
+			if (_log.isDebugEnabled()) {
+				_log.debug(exception, exception);
```

Comment:
I've changed the log to debug because the failing tests were related to logging the exception as an error